### ISSUE TITLE
How comes transforms.Normalize((0.4914, 0.4822, 0.4465), (0.2023, 0.1994, 0.2010)

### ISSUE THREAD

**Issue by xwuaustin** [kuangliu/pytorch-cifar#16]
Forgive my ignorance, I am wondering about the the following code 
transforms.Normalize((0.4914, 0.4822, 0.4465), (0.2023, 0.1994, 0.2010)
Why are these number for (mean, std)? Are they for some mini-batch (how large the batch size) or total dataset?

Thanks?

**Comment by ljk628** [comment]
I am having the same question. It seems the values are different with the ones used in fb.resnet.torch (https://github.com/facebook/fb.resnet.torch/blob/master/datasets/cifar10.lua#L39), Wide ResNet (https://github.com/szagoruyko/wide-residual-networks/blob/master/train.lua#L57) and DenseNet (https://github.com/liuzhuang13/DenseNet/blob/master/datasets/cifar10.lua#L39).

After dividing 255, the correct values are to be
mean = [ 0.49137255,  0.48235294,  0.44666667], std = [ 0.24705882,  0.24352941,  0.26156863].
It would be great to make them consistent with the torch versions for better comparision.

**Comment by kuangliu** [comment]
I computed mean & std by myself. See utils.py `get_mean_and_std`.
You can try different numbers or even remove the normalization. It works fine because of the existence of BN layers.

**Comment by ljk628** [comment]
Thanks for the answer, just find the same question was asked before https://github.com/kuangliu/pytorch-cifar/issues/8 .

**Comment by ghltshubh** [comment]
What about https://github.com/kuangliu/pytorch-cifar/blob/62321904f06202040c919e0fc50b58e34ab26539/main.py#L34

**Comment by ghltshubh** [comment]
> Thanks for the answer, just find the same question was asked before #8 .

What about https://github.com/kuangliu/pytorch-cifar/blob/62321904f06202040c919e0fc50b58e34ab26539/main.py#L34
